src/cpp/shared/town/buildings.cpp

```diff
@@ -99,4 +99,251 @@ char * GetIronfistDwellingName(int faction, int dwelling) {
   }
 
   return "INVALID DWELLING";
-}
\ No newline at end of file
+}
+
+extern BUILDING_CODE gBuildingsToDraw[MAX_FACTIONS][BUILDING_MAX] = {
```

Comment:
Why is this extern?  I don't know what it means to define a variable but also declare it with extern linkage.

Reply:
Confirmed that `extern` isn't required here and removed it.